src/main/java/mops/foren/infrastructure/web/LogoutController.java

```diff
@@ -0,0 +1,18 @@
+package mops.foren.infrastructure.web;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.context.annotation.SessionScope;
+
+import javax.servlet.http.HttpServletRequest;
+
+@Controller
+@SessionScope
+public class LogoutController {
+
+    @GetMapping("/logout")
+    public String logout(HttpServletRequest request) throws Exception {
+        request.logout();
+        return "redirect:/";
```

Comment:
Just like above the redirect should be to /foren

Reply:
ich bin gegen einen Logout auf Get Request. Das ist doch nicht die Art und Weise wie es in der Vorlesung gelehrt wird

Reply:
War nur der Code-Schnipsel aus dem Styleguide, ich hab nicht drüber nachgedacht an der Stelle. Weiß nicht ob das zu 100% verlangt wird den logout auch als GET zu mappen. Ich frag gleich auf jeden Fall.

Reply:
Sollten das definitiv wie im Styleguide vorgeschrieben umsetzen.